tests/python/unittest/test_operator.py

```diff
@@ -4388,13 +4388,14 @@ def test_normal_case():
 
     def test_empty_tensor():
         shape = (2, 3, 0, 4)
-        a = np.array([], dtype=np.int32).reshape(shape)
-        b = mx.nd.array(a, ctx=default_context(), dtype=a.dtype)
-        reps = (2, 4, 6)
+        with mx.np_compat():
```

Comment:
nit: Instead of using the `with` statement, you can decorate this function with `mx.use_np_compat` to avoid indenting the code. This would introduce less difference between two code versions.

Reply:
But decorating the whole function means not testing the version without `np_compat`. Not sure if that's desirable.

Reply:
Empty tensors are not supported anyway without numpy shape semantics. The original test put here was actually not desirable. I'm fine with your change here. It's just a matter of code tidiness.

Reply:
A-ha, so the old test was never guaranteed to pass? In that case, I'll use the decorator, of course.